docs/core/migration/20-21.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Migrate from .NET Core 2.0 to 2.1
+titleSuffix: ""
```

Comment:
Which approach is better ?

The current one ? or change the title to `"Migrate from 2.0 to 2.1"` and keep the docfx suffix.

Current approach makes the title:

"Migrate from .NET Core 2.0 to 2.1 | Microsoft Docs"

second one makes it

"Migrate from 2.0 to 2.1 - .NET Core | Microsoft Docs"

Reply:
I like the current approach.